inspire_matcher/api.py

```diff
@@ -59,11 +59,9 @@ def match(record, config=None):
         config = current_app.config['MATCHER_DEFAULT_CONFIGURATION']
 
     try:
+        index = get_prefixed_index(config['index'])
```

Comment:
sorry, only saw this now, but from https://github.com/inspirehep/inspire-matcher/pull/49/files it looks like you could use `invenio_search.utils.prefix_index` instead.